backend/workflow_manager/endpoint_v2/base_connector.py

```diff
@@ -80,8 +86,14 @@ def get_json_schema(cls, file_path: str) -> dict[str, Any]:
         json.JSONDecodeError: If there is an issue decoding the JSON file.
         """
         try:
-            with open(file_path, encoding="utf-8") as file:
+            if check_feature_flag_status(FeatureFlag.REMOTE_FILE_STORAGE):
+                file_system = FileSystem(FileStorageType.WORKFLOW_EXECUTION)
+                file_storage = file_system.get_file_storage()
+                file = file_storage.read(path=file_path, encoding="utf-8")
```

Comment:
file can be renamed to file_contents or data for better readability.

Reply:
changed